<%
	throw new RuntimeException(new RuntimeException("Testing exception"));
%>